temba/channels/types/playmobile/views.py

```diff
@@ -0,0 +1,39 @@
+
+from smartmin.views import SmartFormView
+
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+from ...models import Channel
+from ...views import ClaimViewMixin
+
+
+class ClaimView(ClaimViewMixin, SmartFormView):
+    class PMClaimForm(ClaimViewMixin.Form):
+        shortcode = forms.CharField(max_length=15, min_length=1, help_text=_("The short code you are connecting"))
+        username = forms.CharField(label=_("Username"), help_text=_("The username for your API account"))
+        password = forms.CharField(label=_("Password"), help_text=_("The password for your API account"))
+
+    form_class = PMClaimForm
+
+    def form_valid(self, form):
+        user = self.request.user
+        data = form.cleaned_data
+        org = user.get_org()
+
+        if not org:  # pragma: no cover
+            raise Exception(_("No org for this user, cannot claim"))
+
+        from .type import PlayMobileType
+
+        config = {
+            PlayMobileType.CONFIG_SHORTCODE: data["shortcode"],
```

Comment:
why is the shortcode should added in the config while we saved it in the address field?

Reply:
Done. Shortcode saved in the address field.